code/game/gamemodes/cult/ritual.dm

```diff
@@ -1,240 +1,70 @@
 #define CULT_ELDERGOD "eldergod"
 #define CULT_SLAUGHTER "slaughter"
 
-/obj/effect/rune/proc/fizzle()
-	if(istype(src,/obj/effect/rune))
-		usr.say(pick("Hakkrutju gopoenjim.", "Nherasai pivroiashan.", "Firjji prhiv mazenhor.", "Tanah eh wakantahe.", "Obliyae na oraie.", "Miyf hon vnor'c.", "Wakabai hij fen juswix."))
-	else
-		usr.whisper(pick("Hakkrutju gopoenjim.", "Nherasai pivroiashan.", "Firjji prhiv mazenhor.", "Tanah eh wakantahe.", "Obliyae na oraie.", "Miyf hon vnor'c.", "Wakabai hij fen juswix."))
-	for (var/mob/V in viewers(src))
-		V.show_message("<span class='warning'>The markings pulse with a small burst of light, then fall dark.</span>", 3, "<span class='warning'>You hear a faint fizzle.</span>", 2)
-	return
-
-/obj/effect/rune/proc/check_icon()
-	if(!SSticker.mode)//work around for maps with runes and cultdat is not loaded all the way
-		var/bits = make_bit_triplet()
-		icon = get_rune(bits)
-	else
-		icon = get_rune_cult(invocation)
-
-/obj/item/tome
-	name = "arcane tome"
-	desc = "An old, dusty tome with frayed edges and a sinister-looking cover."
-	icon_state ="tome"
-	throw_speed = 2
-	throw_range = 5
+/obj/item/melee/cultblade/dagger
+	name = "ritual dagger"
+	desc = "A strange dagger said to be used by sinister groups for \"preparing\" a corpse before sacrificing it to their dark gods."
+	icon = 'icons/obj/cult.dmi'
+	icon_state = "cult_dagger"
+	item_state = "cult_dagger"
 	w_class = WEIGHT_CLASS_SMALL
-	var/scribereduct = 0
-	var/canbypass = 0 //ADMINBUS
-
-/obj/item/tome/accursed
-	name = "accursed tome"
-	desc = "An arcane tome still empowered with a shadow of its former consecration."
-	scribereduct = 30 //faster because it's made by corrupting a bible
-
-/obj/item/tome/imbued //Admin-only tome, allows instant drawing of runes

```

Comment:
i woukd suggest an admin dagger for bus and testing so you can bypass stuff. but thatsme

Reply:
added an adminbus dagger